libs/helm/sidebar/src/lib/hlm-sidebar-inset.ts

```diff
@@ -1,28 +1,20 @@
-import { computed, Directive, inject, input } from '@angular/core';
+import { computed, Directive, input } from '@angular/core';
 import { hlm } from '@spartan-ng/helm/utils';
-
-import { HlmSidebarService } from './hlm-sidebar.service';
-
 import type { ClassValue } from 'clsx';
 
 @Directive({
-	selector: 'main[hlmSidebarInset]', // Restrict to main elements only
-
+	selector: 'main[hlmSidebarInset]',
 	host: {
 		'data-slot': 'sidebar-inset',
 		'[class]': '_computedClass()',
 	},
 })
 export class HlmSidebarInset {
-	private readonly _sidebarService = inject(HlmSidebarService);
-
 	public readonly userClass = input<ClassValue>('', { alias: 'class' });
 	protected readonly _computedClass = computed(() =>
 		hlm(
 			'bg-background relative flex w-full flex-1 flex-col',
-			this._sidebarService.variant() === 'inset' &&
-				'min-h-[calc(100svh-theme(spacing.4)))] md:m-2 md:rounded-xl md:shadow-sm',
-			this._sidebarService.open() ? 'md:ml-0' : 'md:ml-2',
+			'md:peer-data-[variant=inset]:m-2 md:peer-data-[variant=inset]:ml-0 md:peer-data-[variant=inset]:rounded-xl md:peer-data-[variant=inset]:shadow-sm md:peer-data-[variant=inset]:peer-data-[state=collapsed]:ml-2',
```

Comment:
@MerlinMoos compared the inset classes with shadcn/ui and this is used. 
Is there a specific use case for your implementation?